sdk/core/azure-core/src/http/curl/curl_transport.cpp

```diff
@@ -1,52 +1,51 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// SPDX-License-Identifier: MIT
 
-#include <curl_client.hpp>
-#include <http/http.hpp>
+#include "azure.hpp"
+#include "http/curl/curl.hpp"
+#include "http/http.hpp"
 
-#include <iostream>
-#include <memory>
+#include <string>
 
 using namespace Azure::Core::Http;
-using namespace std;
 
-std::unique_ptr<Response> CurlClient::Send()
+CurlTransport::CurlTransport() : Transport() { m_pCurl = curl_easy_init(); }
```

Comment:
I don't think the ctor/dtor call curl_easy_init/curl_easy_cleanup. These functions are called once per I/O operation; not once per Transport instance.
So, call them at the top/bottom of Send.